core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -434,7 +434,14 @@ public void closed(Status status, Metadata trailers) {
           trailers = new Metadata();
         }
       }
-      final Status savedStatus = status;
+      ClientTransport transport = null;
+      try {
+        transport = transportFuture.get();
+      } catch (Exception e) {
+        // Ignore the exception, and keep transport as null.
+      }
+      final Status savedStatus = (transport == null
+          ? status : status.augmentDescription("transport=" + transport));
```

Comment:
I don't think this is appropriate. A status description should be _exactly_ what was returned by the server (unless the server did not provide one).

Instead, we could maybe add another field to Status that is local-only.